core/domain/summary_services_test.py

```diff
@@ -252,12 +253,21 @@ def setUp(self):
         self.login(self.ADMIN_EMAIL, is_super_admin=True)
         response = self.testapp.get('/admin')
         csrf_token = self.get_csrf_token_from_response(response)
-        self.post_json(
-            '/adminhandler', {
-                'action': 'reload_exploration',
-                'exploration_id': '2'
-            }, csrf_token=csrf_token)
-        self.logout()
+
+        def mock_get_filename_with_dimensions(filename, unused_exp_id):
+            return exp_services.regenerate_image_filename_using_dimensions(
+                filename, 490, 120)
+
+        with self.swap(
```

Comment:
You can get rid of this by just upgrading the demo exploration 2.

Load it in a browser in the dev server, and then download it.

It's OK to file a cleanup issue for this and do it later, but make sure the cleanup issue has full and clear instructions.

Reply:
Okay. Will file it.

Reply:
I was about to create the issue, but I don't exactly know the procedure of upgrading it(you asked about writing clear instructions)

Reply:
Go to /admin and reload the exploration.
Go to the /create page for that exploration and go to the history tab.
Download the exploration.
Replace the YAML file and assets accordingly in the data/ folder.
Go back to /admin, reload the exploration, and check that it comes out right.

Reply:
Thanks